scripts/quick-deploy-service.ts

```diff
@@ -0,0 +1,393 @@
+  uniqueID: string,
+): void {
+  const servicePrefix = serviceId.split('-service')[0];
+
+  const manifestUploadCommand = `yarn util:load-vars mosaic hosting manifest upload -i ${serviceId} -p ./services/${servicePrefix}/service/mosaic-hosting-deployment-manifest.yaml -n ${serviceId}-manifest-${uniqueID}`;
+
+  console.log(
+    `\nRunning Manifest Upload command:\n${chalk.green(
+      manifestUploadCommand,
+    )}\n`,
+  );
+
+  execSync(manifestUploadCommand, { stdio: 'inherit' });
+}
+
+function initiateDeployment(
+  serviceId: 'media-service' | 'catalog-service' | 'entitlement-service',
+  dockerImageTag: string,
+  uniqueID: string,
+): void {
+  const deployCommand = `yarn util:load-vars mosaic hosting service deploy -i ${serviceId} -t ${dockerImageTag} ${
+    serviceId === 'media-service' ? '-p media-workflows@1.0.0' : ''
+  } -m ${serviceId}-manifest-${uniqueID} -n ${serviceId}-deployment-${uniqueID}`;
+
+  console.log(`\nRunning Deploy command:\n${chalk.green(deployCommand)}\n`);
+
+  execSync(deployCommand, {
+    stdio: 'inherit',
+  });
+}
+
+/**
+ * Prints the URL to the Mosaic Admin Portal to open up the Custom Service details station.
+ */
+function printAdminPortalURL(serviceDefinitionId: string): void {
+  const idServiceAuthBaseUrl = from(process.env)
+    .get('ID_SERVICE_AUTH_BASE_URL')
+    .required()
+    .asString();
+
+  const environmentId = from(process.env)
+    .get('ENVIRONMENT_ID')
+    .required()
+    .asString();
+
+  const adminPortalBaseUrl = idServiceAuthBaseUrl.includes('.eu.')
+    ? 'https://admin.service.eu.axinom.com'
+    : idServiceAuthBaseUrl.includes('.test.')
+    ? 'https://admin.service.test.axtest.net'
+    : idServiceAuthBaseUrl.includes('.cb.')
+    ? 'https://admin.service.cb.axtest.net'
+    : 'https://admin.service.dev.axtest.net';
+
+  const adminPortalURL = `${adminPortalBaseUrl}/env/${environmentId}/services/ax-hosting-service/config/customizable-services/${serviceDefinitionId}`;
+
+  console.log(
+    `\nSee more details at the Mosaic Admin Portal:\n${chalk.green(
+      adminPortalURL,
+    )}\n`,
+  );
+}
+
+async function main(): Promise<void> {
+  try {
+    const answers = await prompt([
+      {
+        type: 'select',
+        name: 'serviceId',
+        message: 'Select the service to deploy',
+        choices: [
+          { title: 'Media Service', value: 'media-service' },
+          { title: 'Catalog Service', value: 'catalog-service' },
+          { title: 'Entitlement Service', value: 'entitlement-service' },
+        ],
+      },
+      {
+        type: 'text',
+        name: 'dockerImageTag',
+        message:
+          'If you want to deploy a specific docker image tag that is already pushed to the registry, enter that image tag here. Leave this blank if you you are not sure.',
+        initial: '',
+      },
+      {
+        type: 'confirm',
+        name: 'deploy',
+        message: 'Confirm the deployment of the selected service?',
```

Comment:
this was a bit confusing in the video tbh
wasn't sure what it was configuring and was guessing with this to `yes` it would just build and then ask again before it start the deployment. But from code, I see now, that this is essentially an "are you sure you want to start deploying?" prompt. Which means, we should probably also phrase it like this?